Dockerfile

```diff
@@ -1,7 +1,27 @@
 FROM python:3-alpine
+FROM node:carbon-alpine AS build
+
+FROM ubuntu
+RUN apt-get update && apt-get install -y python3-pip
+
 ENV PYTHONUNBUFFERED 1
 RUN mkdir /code
 WORKDIR /code
-ADD requirements.txt /code/
+COPY ./ /code/
+
+
+
+RUN apt-get update
+RUN apt-get install curl gettext -y
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN apt-get install nodejs build-essential -y
+RUN apt-get install binutils libproj-dev gdal-bin -y
```

Comment:
It would be more optimized if you combined all of the packages installed on these `apt-get install` lines into one line, that runs after the `curl` command to install the nodesource repos.